addons/point_of_sale/static/src/js/models.js

```diff
@@ -443,9 +443,11 @@ function openerp_pos_models(instance, module){ //module is instance.point_of_sal
             var self = this;
             var def  = new $.Deferred();
             var fields = _.find(this.models,function(model){ return model.model === 'res.partner'; }).fields;
+            var domain = _.find(this.models, function(model){ return model.model === 'res.partner'; }).domain.slice(0);
+            domain.push(['write_date','>',this.db.get_partner_write_date()]);
```

Comment:
why slice(0) ?

Reply:
to make a copy of the list. This function will be called many times, if slice(0) is not set, 
- the domain will be changed (not desired),
- the domain become longer each time.

Ref : https://stackoverflow.com/questions/15722433/javascript-copy-array-to-new-array

regards.